src/pages/BorrowerProfile/FundedBorrowerProfile.vue

```diff
@@ -231,7 +232,25 @@ export default {
 					name: 'description',
 					content: `A loan helped ${this.loan.use}`,
 				}
-			]
+			].concat(
```

Comment:
This is fine as a minimum but I think the funded profile should have the same meta fields as the borrower profile. For example 'og:type', 'og:image', 'twitter:image', 'twitter:title'